x-pack/plugins/security_solution/public/timelines/components/timeline/body/index.test.tsx

```diff
@@ -84,7 +83,6 @@ jest.mock('../../../../common/lib/kibana', () => {
         },
       },
     }),
-    useGetUserSavedObjectPermissions: jest.fn(),
```

Comment:
This hook no longer exists, it was renamed to `useGetUserCasesPermissions` outside of this PR.